tests/unit/src/unit/TestMatch.hx

```diff
@@ -304,109 +308,112 @@ class TestMatch extends Test {
 			case [_, true, _]:
 		}));
-		//var x:Null<Bool> = true;
-		//eq("Unmatched patterns: null", getErrorMessage(switch x {
-			//case true:
-			//case false:
-		//}));
-		//var t:Null<Tree<String>> = null;
-		//eq("Unmatched patterns: null", getErrorMessage(switch t {
-			//case Leaf(_):
-			//case Node(_):
-		//}));
+		// var x:Null<Bool> = true;
+		// eq("Unmatched patterns: null", getErrorMessage(switch x {
+		// case true:
+		// case false:
+		// }));
+		// var t:Null<Tree<String>> = null;
+		// eq("Unmatched patterns: null", getErrorMessage(switch t {
+		// case Leaf(_):
+		// case Node(_):
+		// }));
 	}
 
 	function testInvalidBinding() {
-		eq("Variable y must appear exactly once in each sub-pattern", getErrorMessage(switch(Leaf("foo")) {
+		eq("Variable y must appear exactly once in each sub-pattern", getErrorMessage(switch (Leaf("foo")) {
 			case Leaf(x) | Leaf(y):
 		}));
-		eq("Variable y must appear exactly once in each sub-pattern", getErrorMessage(switch(Leaf("foo")) {
+		eq("Variable y must appear exactly once in each sub-pattern", getErrorMessage(switch (Leaf("foo")) {
 			case Leaf(x) | Leaf(x) | Leaf(y):
 		}));
-		eq("Variable x must appear exactly once in each sub-pattern", getErrorMessage(switch(Leaf("foo")) {
+		eq("Variable x must appear exactly once in each sub-pattern", getErrorMessage(switch (Leaf("foo")) {
 			case Leaf(x) | Leaf(x) | Leaf(_):
 		}));
-		eq("Variable l must appear exactly once in each sub-pattern", getErrorMessage(switch(Leaf("foo")) {
-			case Node(l = Leaf(x),_) | Node(Leaf(x), _):
+		eq("Variable l must appear exactly once in each sub-pattern", getErrorMessage(switch (Leaf("foo")) {
+			case Node(l = Leaf(x), _) | Node(Leaf(x), _):
 		}));
-		eq("Variable l is bound multiple times", getErrorMessage(switch(Leaf("foo")) {
+		eq("Variable l is bound multiple times", getErrorMessage(switch (Leaf("foo")) {
 			case Node(l = Leaf(l), _):
 		}));
-		eq("String should be unit.Tree<String>", getErrorMessage(switch(Leaf("foo")) {
+		eq("String should be unit.Tree<String>", getErrorMessage(switch (Leaf("foo")) {
 			case Node(l = Leaf(_), _) | Leaf(l):
 			case _:
 		}));
 	}
 
 	function testNullPattern() {
 		var i:Null<Int> = null;
-		var r = switch(i) {
+		var r = switch (i) {
 			case 1: 1;
 			case null: 2;
 			case _: 3;
 		}
 		eq(2, r);
 
 		// this should not compile because the argument is not explicitly Null
-		//var e = EConst(null);
-		//var r = switch(e) {
-			//case EConst(null): 1;
-			//case _: 2;
-		//}
+		// var e = EConst(null);
+		// var r = switch(e) {
+		// case EConst(null): 1;
+		// case _: 2;
+		// }
 
 		var t:Null<Tree<String>> = null;
-		var r = switch(t) {
+		var r = switch (t) {
 			case Leaf(_): 1;
 			case null if (i != null): 2;
 			case null: 3;
 			case Node(_): 4;
 		}
 		eq(r, 3);
 
-		var e1 = macro if (1) 2;
-		var e2 = macro if (1) 2 else 3;
+		var e1 = macro if (1)
+			2;
+		var e2 = macro if (1)
+			2
+		else
+			3;
```

Comment:
Is this some haxe formatter bug? @AlexHaxe

Reply:
I should really pay more attention when saving these older files...

Reply:
Well it's a lot of noise but if we have a formatting config that fits what we want for haxe code it shouldn't matter much eventually

Reply:
yes, that's a bug, it should've stayed in one line, but `macro` keyword threw it off, treating it like a standalone if.
should be fixed soon.